static/styles/rewards/gift-cards.css

```diff
@@ -1,193 +1,178 @@
-  display: none;
+.card-error {
+  text-align: center;
+  padding: 0 12px;
 }
 
-.gift-cards-wrapper {
+#offered-card {
   display: flex;
-  flex-direction: row;
-  overflow-x: auto;
-  max-width: 563px;
-  width: 100vw;
-  transform: rotateX(180deg);
-  /* Because of rotateX, some properties like padding-bottom, align-items work in reverse. */
-  align-items: end;
-  padding-bottom: 12px;
-}
-.gift-cards-wrapper.center {
-  justify-content: center;
-}
-.gift-cards-wrapper .gift-card {
-  transform: rotateX(180deg);
 }
 
-.purchased {
-  flex-direction: column;
-  align-items: center;
+@media screen and (max-width: 640px) {
+  .card-section {
+    flex-direction: column;
+    text-align: center;
+  }
+  .card-section .details {
+    margin-left: 0;
+  }
 }
 
-.gift-card {
-  background-color: var(--background-color-default);
-  margin-right: 10px;
-  padding: 12px 6px;
-  text-align: center;
-  min-width: 168px;
-  width: 168px;
-  height: fit-content;
-}
-.redeem-code {
-  width: 160px;
-  overflow-wrap: break-word;
-  text-transform: none;
-}
-.redeem-code p {
+#redeem-code p {
   text-transform: none;
+  margin: 2px 0;
 }
 
-.gift-card h3 {
-  color: #fff;
+.card-section h3 {
+  padding: 8px;
+  text-align: center;
 }
 
-.gift-card div.buttons {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-evenly;
-  margin: 10px 0;
-}
-.gift-card div.buttons span {
+.card-section .details {
   display: flex;
   flex-direction: column;
-  align-items: center;
-  justify-content: center;
-  padding: 6px;
+  margin-left: 24px;
+  row-gap: 12px;
+  flex: 1;
 }
 
-.gift-card div.buttons button {
-  background-color: #80808020;
-  border: none;
-  color: #fff;
-  cursor: pointer;
-}
-.gift-card img {
+.card-section img {
   box-sizing: border-box;
   width: 100%;
   height: auto;
   border: 2px solid #80808020;
-  max-width: 168px;
-  height: 110px;
+  max-width: 200px;
 }
 
-.gift-card .pricing {
+.card-section .pricing {
   display: flex;
   flex-direction: column;
 }
-.gift-card .pricing .available {
-  color: #fff;
+.card-section .pricing .available {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
 }
-.gift-card .pricing div {
+.card-section .pricing .available div .currency {
   display: flex;
   flex-direction: row;
-  justify-content: space-between;
+  justify-content: center;
 }
-.redeem-info-wrapper {
-  position: absolute;
-  z-index: 1000;
-  top: 0;
-  left: 0;
-  width: 100%;
-  height: 100%;
-  background-color: var(--background-color-default);
+.card-section .pricing .available div .currency div {
+  font-size: 8px;
+}
+.card-section .pricing .available div .amount {
+  font-size: 24px;
+  color: #fff;
 }
 .redeem-info {
-  position: fixed;
-  top: 50%;
-  left: 50%;
-  transform: translate(-50%, -50%);
-  max-width: 563px;
-  padding: 24px;
-  border-collapse: collapse;
-  border-color: gray;
-  overflow-y: auto;
-  z-index: 1;
+  border-color: var(--light-mode-border-color);
```

Comment:
I think this might be wrong. You might need to put dark mode themes as default and the build script will generate light mode styles automatically.

Reply:
I have removed it. It is not needed anymore.